packages/teams-js/src/public/constants.ts

```diff
@@ -104,3 +105,18 @@ export enum ChannelType {
 }
 
 export const errorNotSupportedOnPlatform: SdkError = { errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM };
+
+export const minAdaptiveCardVersion: AdaptiveCardVersion = { majorVersion: 1, minorVersion: 5 };
+export function isHostAdaptiveCardSchemaVersionUnsupported(
```

Comment:
Apologies for not noticing this sooner, but should this be moved to the same file as the compareSDKVersions function and marked as @hidden? I don't think app developers are supposed to know about it or call it, and it's also not a constant

Reply:
Moved `isHostAdaptiveCardScehmaVersionUnsupported` to `utils.ts` and added `@hidden` tag to it.